build-logic/performance-testing/src/main/kotlin/gradlebuild/performance/tasks/BuildCommitDistribution.kt

```diff
@@ -206,7 +206,7 @@ abstract class BuildCommitDistribution @Inject internal constructor(
             ":distributions-full:binDistributionZip",
             ":tooling-api:installToolingApiShadedJar",
             "-PtoolingApiShadedJarInstallPath=" + commitDistributionToolingApiJar.get().asFile.absolutePath,
-            "-Porg.gradle.java.installations.paths=${getJavaHomeFor(11)},${getJavaHomeFor(17)}",
+            "-Dorg.gradle.java.installations.paths=${getJavaHomeFor(11)},${getJavaHomeFor(17)}",
```

Comment:
💅 AFAIU we should set this with `-P` and `-D` here as we may build versions here that do not yet have the relevant changes applied to them.

Reply:
I've updated this to use both `-D` and `-P`.